### PR TITLE
fix(copy): mistype on fw update screen and missing translation added

### LINKED ISSUE #2898: Mistype on the firmware update screen
**Describe the bug**
Mistype on the firmware update screen in Suite (the one asking to reconnect the device). “It” instead of “In”.

**Info:**
 - Suite Version: app, commit (?)
 - OS: Mac OS

**Screenshots:**
image

### PR DESCRIPTION
**Pull request by mroz22**
should fix #2898

### CODE REVIEW

File: packages/suite/src/components/firmware/ReconnectInNormal.tsx [hunk 1]
```diff
@@ -59,8 +59,8 @@ const Body = () => {
                     <Translation id="TR_DISCONNECT_YOUR_DEVICE" />
                 </H2>
                 <P>
-                    Device is in unexpected model. In the next step you will be asked to reconnect
-                    your device normally.
+                    <Translation id="TR_DEVICE_IS_IN_UNEXPECTED_MODE" />
+                    <Translation id="FIRMWARE_RECONNECT_NORMAL_NEXT_STEP_DESC" />
```
Comment (keraf) [inline packages/suite/src/components/firmware/ReconnectInNormal.tsx]: Not sure why there are two translations in the same paragraph? Why not one? And there's also a dot missing in the first sentence.

**Review by keraf — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/firmware/ReconnectInNormal.tsx (hunk 1 above)
Comment (mroz22) [reply packages/suite/src/components/firmware/ReconnectInNormal.tsx]: That unexpected mode strig appears two times in the project. So initially I wanted to reuse it, but later I realized that the other occurence was native. I'd say it is no harm to have these two splitted. 
Added the dot.

Comment (keraf) [reply packages/suite/src/components/firmware/ReconnectInNormal.tsx]: Hmm, in this change it looks like `TR_DEVICE_IS_IN_UNEXPECTED_MODE` has been added. In any case, I think it makes more sense to me to have the whole paragraph in a single translation.

Comment (mroz22) [reply packages/suite/src/components/firmware/ReconnectInNormal.tsx]: :dagger: changes done

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**